src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -0,0 +1,652 @@
+# This file collects all methods that have been updated to work with parabolic systems of equations
```

Comment:
`@muladd` stuff?

Reply:
Done in 7b74f19884a553c48ac2f84c97efe7a927d2a894